src/Parallel/Tags/InputSource.hpp

```diff
@@ -0,0 +1,36 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <string>
+
+#include "DataStructures/DataBox/Tag.hpp"
+#include "Options/Options.hpp"
+
+namespace Parallel::OptionTags {
+/// \ingroup OptionTagsGroup
+/// An input file tag tag specifying the yaml input-file source itself,
+/// so that the yaml input-file string can be embedded in H5 output.
+struct InputSource {
+  using type = std::string;
+  static constexpr Options::String help = {
+      "DO NOT SPECIFY THIS OPTION (enables embedding input yaml in H5 "
+      "output)."};
+};
+}  // namespace Parallel::OptionTags
+
+namespace Parallel::Tags {
+/// \ingroup DataBoxTagsGroup
+/// A tag storing the input file yaml input source passed in at runtime, so that
+/// it can be accessed and written to files.
+struct InputSource : db::SimpleTag {
+  using type = std::string;
```

Comment:
and why is this a `std::string` while `Options::Tags::InputSource` is a `std::vector<std::string>` ?

Reply:
After my most recent fixups, this tag now uses std::vector<std::string>